airbyte-workers/src/main/java/io/airbyte/workers/normalization/DefaultNormalizationRunner.java

```diff
@@ -68,7 +71,12 @@ public boolean normalize(long jobId, int attempt, Path jobRoot, JsonNode config,
     IOs.writeFile(jobRoot, WorkerConstants.CATALOG_JSON_FILENAME, Jsons.serialize(catalog));
 
     try {
-      process = pbf.create(jobId, attempt, jobRoot, NORMALIZATION_IMAGE_NAME, "run",
+      final String image_name;
```

Comment:
This is python naming

you can write
```
final String imageName = useDev... ? DEV_NORMALIZATION_IMAGE_NAME :  NORMALIZATION_IMAGE_NAME
```

Reply:
Oops, you're right, I guess I was caught in python world while working/testing this...